app/default_overrides.go

```diff
@@ -263,9 +267,9 @@ func DefaultConsensusConfig() *tmcfg.Config {
 
 	cfg.Mempool.TTLNumBlocks = 12
 	cfg.Mempool.TTLDuration = 75 * time.Second
-	cfg.Mempool.MaxTxBytes = 7_897_088
-	cfg.Mempool.MaxTxsBytes = 39_485_440
-	cfg.Mempool.Version = "v1" // prioritized mempool
+	cfg.Mempool.MaxTxBytes = 2 * mebibyte
+	cfg.Mempool.MaxTxsBytes = 80 * mebibyte
+	cfg.Mempool.Version = "v2" // prioritized mempool
```

Comment:
```suggestion
	cfg.Mempool.Version = "v2" // Content Addressable Transaction (CAT) mempool
```

Reply:
Confirmed this is the CAT mempool. Ref: https://github.com/celestiaorg/celestia-core/blob/ca0c9d7c93a4f66fc2257ff345cb690359060233/consensus/common_test.go#L422-L424

https://github.com/celestiaorg/celestia-core/blob/ca0c9d7c93a4f66fc2257ff345cb690359060233/consensus/common_test.go#L32